crates/sui-benchmark/Cargo.toml

```diff
@@ -35,7 +35,7 @@ sui-config = { path = "../sui-config" }
 sui-types = { path = "../sui-types" }
 
 move-core-types = { git = "https://github.com/move-language/move", rev = "7733658048a8bc80e9ba415b8c99aed9234eaa5f", features = ["address20"] }
-narwhal-node = { git = "https://github.com/MystenLabs/narwhal", rev = "f50af9eb40130fb404a678e38d0671fc33f60318", package = "node" }
+narwhal-node = { git = "https://github.com/punwai/narwhal-1", rev = "172cf0e36a9a1b357cb31df645086de03deffecf", package = "node" }
```

Comment:
Lets not do this and instead land the changes in the narwhal repo and use a commit from the narwhal repo.

Reply:
Yep, I did this to test, the PR should merge in a few minutes.

Reply:
ah didn't realize that was the case :)